components/html-block/html-block.js

```diff
@@ -3,6 +3,43 @@ import { css, LitElement } from 'lit-element/lit-element.js';
 import { htmlBlockMathRenderer } from '../../helpers/mathjax.js';
 import { requestInstance } from '../../mixins/provider-mixin.js';
 
+let contextPromise;
+
+const getContext = () => {
+	if (contextPromise) return contextPromise;
+
+	contextPromise = new Promise(async resolve => { // eslint-disable-line no-async-promise-executor
+		const context = JSON.parse(document.documentElement.getAttribute('data-hb-context'));
```

Comment:
I recently added a new feature to ifrau to sync html element data attributes automatically. For FRAs to take advantage of this particular feature within this component, they'll need to update ifrau to v0.36.0.

Reply:
Also note: This context object isn't set in the LMS yet. I'm just trying to lay down the groundwork in the components first.

Reply:
What does this yield if the data attribute doesn't exist?  Should we be defaulting to an empty object?

Reply:
Yeah, I haven't updated this PR since I made the changes to the HTML editor context stuff, but I was intending on propagating those changes here (or wherever we move the context fetching).